src/main/java/org/folio/search/controller/FolioTenantController.java

```diff
@@ -0,0 +1,33 @@
+package org.folio.search.controller;
+
+import lombok.extern.log4j.Log4j2;
+import org.folio.search.service.KafkaAdminService;
+import org.folio.spring.FolioExecutionContext;
+import org.folio.spring.controller.TenantController;
+import org.folio.spring.liquibase.FolioSpringLiquibase;
+import org.folio.tenant.domain.dto.TenantAttributes;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@Log4j2
+@RequestMapping(value = "/_/")
+@RestController("folioTenantController")
+public class FolioTenantController extends TenantController {
+
+  private final KafkaAdminService kafkaAdminService;
+
+  public FolioTenantController(
+    FolioSpringLiquibase folioSpringLiquibase,
+    FolioExecutionContext context,
+    KafkaAdminService kafkaAdminService) {
+    super(folioSpringLiquibase, context);
+    this.kafkaAdminService = kafkaAdminService;
+  }
+
+  @Override
+  public ResponseEntity<String> postTenant(TenantAttributes tenantAttributes) {
+    kafkaAdminService.createKafkaTopics();
```

Comment:
Why do we need to create topics in mod-search?

Reply:
Ideally, producer and consumer should be independent and both of them should be able to create topics using common configuration. For example, if mod-search started earlier than mod-inventory-storage - we don't want to see tons of logs with unknown_topic_or_partition errors

Reply:
Make sense, thanks.